### PR TITLE
Updating documentation

### PR DESCRIPTION
**Pull request by Peque**
A work-in-progress branch to update osBrain documentation.

### CODE REVIEW

File: docs/source/about.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+taken during the development process:
+
+#. **Isolation of agents**; creating separate system processes to avoid shared
+   memory and any problems derived from multi-threading development.
+#. **Implementation of message passing**; making use of the modern, efficient
+   and flexible `ØMQ <http://zeromq.org/>`_ library.
+#. **Ease of configuration/deployment**; making use of the very convenient,
+   well implemented and documented `Pyro4 <https://pythonhosted.org/Pyro4/>`_
+   package.
+#. **Separation from the trading platform**; what it started as a basic
```
Comment (Flood1993) [inline docs/source/about.rst]: I think rewording this sentence like this sounds more natural: `what started as a basic architecture for implementing a real-time automated-trading platform, ended-up being a general-purpose multi-agent system architecture.`.

File: docs/source/developers.rst [hunk 2]
```diff
@@ -1,13 +1,47 @@
...
+Workflow
+========
+
+We are happy you like osBrain and we would love to receive contributions from
+you! Note that contributions do not necessarily need to include code: you can
+help `telling us what problems you are having with osBrain <https://github.com/opensistemas-hub/osbrain/issues>`_
+or suggesting improvements to the documentation.
+
+If you would like to help us with code, proceed to fork the project, make the
+changes you want and then submit a pulll request to start a discussion. Take
```
Comment (Flood1993) [inline docs/source/developers.rst]: Three `l`'s in `pull request`.

File: docs/source/introduction.rst [hunk 3]
```diff
@@ -2,135 +2,121 @@
...
+>>> agent.log_info('Hello world')
 
-.. literalinclude:: ../../examples/hello_world.py
+The method ``log_info()`` is implemented in
+:class:`osbrain.Agent <osbrain.agent.Agent>` so,
+when this method is called from the proxy, this call is actually being
+serialized to the remote running agent and gets executed there. The return
+value, if any, is then serialized back and returned by the proxy. So basically
+so get the impression of being working with a local object while your code is
+executed remotelly.
```
Comment (Flood1993) [inline docs/source/introduction.rst]: An extra `l` in `remotely`.

File: docs/source/introduction.rst [hunk 4]
```diff
@@ -2,135 +2,121 @@
...
-#. osBrain uses `pickle <https://docs.python.org/library/pickle.html>`_ for
-   serialization, which means that the system performance depends on this
-   package.
-#. ØMQ transport is IPC by default. It can be changed to TCP globally or
-   configured specifically for each bind. Note, however, that in this case
-   the network may have a great impact on performance. TCP must be used, of
-   course, for distributed systems.
+The code above creates (and registers) three different agents in a name server
+and then creates, through the name server proxy, a proxy to one of those agents
+simply using its alias. Then it uses the agent proxy to remotelly call a
```
Comment (Flood1993) [inline docs/source/introduction.rst]: An extra l in remotely.

File: docs/source/introduction.rst [hunk 5]
```diff
@@ -2,135 +2,121 @@
...
 
+.. index:: name server
 
-.. index:: performance
+The name server
+===============
 
-Performance
-===========
+A name server is just like another agent, so it runs independently, but with
```
Comment (Flood1993) [inline docs/source/introduction.rst]: Perhaps replace `another` with `any other`?

**Comment by Peque** [comment]
@Flood1993 I included all your suggestions. Thank you very much. :blush:

**Pull request merged**